app/ui.js

```diff
@@ -16,12 +16,14 @@ CONTROLLERS
 */
 const { promptNewCustomer } = require('./controllers/customerCtrl')
 const promptActivateCustomer = require('./controllers/activateCustomerCtrl')
+const promptAddCustomerProduct = require('./controllers/addCustomerProductCtrl');
 const { promptPaymentType } = require('./controllers/addPaymentTypeCtrl')
 
 /*
 MODELS
 */
 const getCustomers = require('./models/getCustomers');
+const addCustomerProduct = require('./models/AddCustomerProduct');
```

Comment:
I think this should be AddCustomerProduct.  I had accidentally wrote getCustomers instead of GetCustomer.  I'll fix that.

Reply:
Wait, I'm wrong here.  You had named it correctly. Don't worry about it.  Carry on.

Reply:
👍